deps/verifier/src/eventlog/mod.rs

```diff
@@ -24,10 +24,7 @@ impl FromStr for AAEvent {
 
     fn from_str(input: &str) -> Result<Self> {
         let input_trimed = input.trim_end();
-        let sections: Vec<&str> = input_trimed.split(' ').collect();
-        if sections.len() != 3 {
-            bail!("Illegal AA event entry format. Should be `<domain> <operation> <content>`");
-        }
+        let sections: Vec<&str> = input_trimed.splitn(3, ' ').collect();
```

Comment:
I think this will panic later if the input string is "one" or "one two", so I would keep the length check